py-polars/polars/dataframe/frame.py

```diff
@@ -3390,8 +3390,8 @@ def write_parquet(
         compression: ParquetCompression = "zstd",
         compression_level: int | None = None,
         statistics: bool = False,
-        row_group_size: int | None = None,
-        data_page_size: int | None = None,
+        row_group_size: int | None = 512 ** 2,
+        data_page_size: int = 1024 ** 2,
```

Comment:
In the Python interface, I find it a little confusing that this is `None` by default. In any case, `pyarrow` and `polars` use this value.